store/commitment/store.go

```diff
@@ -460,6 +461,10 @@ loop:
 		}
 	}
 
+	if err := importer.Close(); err != nil {
```

Comment:
This change isn't correct because importer gets replaced in multiple loops, hence I'd keep it as it was before.